docs/docsite/rst/playbook_guide/playbooks_handlers.rst

```diff
@@ -114,6 +114,39 @@ Each handler should have a globally unique name. If multiple handlers are define
 
 There is only one global scope for handlers (handler names and listen topics) regardless of where the handlers are defined. This also includes handlers defined in roles.
 
+Controlling the notifing of handlers by defining "changed"
+----------------------------------------------------------
+
+By using ``changed_when`` keyword, you define when a task has "changed" status. This means you directly determine when ``notify`` is triggered so it executes the handlers.
+
+In following example it is ensured that the handler restarts the service every time the new configuration gets copied:
+
+.. code-block:: yaml
+
+    - name: Copy httpd configuration
+        ansible.builtin.copy:
+          src: ./new_httpd.conf
+          dest: /etc/httpd/conf/httpd.conf
+        # The task is always reported as changed
+        changed_when: True
+        notify: Restart apache
+
+Also you can execute handler for example based on command return value:
+
+.. code-block:: yaml
+
+    tasks:
+      - name: Task with handled command result
+        ansible.builtin.command: /bin/utils/check_command
+        register: result
+        changed_when: result.rc == 2 # something is not properly initialized
+        notify: Initialize files
+
+    handlers:
+      - name: Initialize files
+        ...
```

Comment:
This next example is OK but I think it is a little superfluous here. The example isn't consistent with the other examples, in that it isn't really about a service. This fits better with the section that you link to at the end so I'd just leave it out and keep it brief.